invenio_jobs/services/services.py

```diff
@@ -215,8 +220,9 @@ def read(self, identity, job_id, run_id):
             self, identity, run_record, links_tpl=self.links_item_tpl
         )
 
+    @with_job_context()
```

Comment:
Add a comment to highlight the order of the decorators is important tot not skipp any logging